src/features/classification/Classification.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import * as React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import type { InjectIntlProvidedProps } from 'react-intl';
```

Comment:
nit: Should we group import types together?

Reply:
Yikes, this got away from me. Yes but I'll combine it with the shift to FormattedDate.